.env.example

```
@@ -35,9 +56,20 @@ CREDENTIAL_ENCRYPTION_ALLOW_DEV_KEY=1
-# Required for production: admin auth + API
+# Required: admin auth + API
 # ---------------------------------------------------------------------
+# With SECRET_PROVIDER=env, the *_SECRET / *_SECRET_ARN vars hold the
+# NAME of another env var whose value is the actual secret. The matched
+# env var must then be defined below. Two reasons for the indirection:
+# (1) production uses the same var name pointing at a real ARN/name;
+# (2) the dev value is co-located with its lookup key so future readers
+# can trace the chain in one file.
 ADMIN_EMAIL=admin@example.com
+ADMIN_PASSWORD_SECRET=ADMIN_PASSWORD_DEV
+ADMIN_PASSWORD_DEV=LocalDev!Pass123
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ADMIN_PASSWORD_DEV key should go before the ADMIN_PASSWORD_SECRET key

(UnorderedKey)